docker-compose.yml

```diff
@@ -34,13 +34,45 @@ services:
     ports:
       - "26381:26381"
 
+  cluster_master_1:
+    build: docker/cluster/master_1
```

Comment:
Let's use the container_name clause for each docker (even though we're not elsewhere). Locally, this makes debugging significantly easier.

Reply:
Added